### PR TITLE
add example for code mounting with cdk

### PR DESCRIPTION
**Pull request by lukqw**
This PR adds a new section to ECS docs, which outlines how to achieve code mounting with cdk.

### CODE REVIEW

File: content/en/user-guide/aws/ecs/index.md [hunk 1]
```diff
@@ -229,3 +231,25 @@ ecs_client.register_task_definition(
     volumes=[{"host": {"sourcePath": "/host/path"}, "name": "test-volume"}],
 )
 ```
+
+### CDK example
+
+The same functionality can be achieved in CDK following this example:
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:237]: Small wording suggestion

```suggestion
The same functionality can be achieved with the AWS CDK following this (Python) example:
```

File: content/en/user-guide/aws/ecs/index.md [hunk 2]
```diff
@@ -229,3 +231,25 @@ ecs_client.register_task_definition(
...
+
+container = task_def.add_container(...)
+
+container.add_mount_points(
+        ecs.MountPoint(
+                container_path="/container/path",
+                source_volume="test-volume",
+        ),
+)
+```
```
Comment (simonrw) [inline content/en/user-guide/aws/ecs/index.md:255]: Are you mad, do you use 8 space indenting?!?!?! 😂 

```suggestion
```python
task_definition = ecs.TaskDefinition(
    ...
    volumes=[
        ecs.Volume(name="test-volume", host=ecs.Host(source_path="/host/path"))
    ]
)

container = task_def.add_container(...)

container.add_mount_points(
    ecs.MountPoint(
        container_path="/container/path",
        source_volume="test-volume",
    ),
)
```
```

**Review by simonrw — COMMENTED** [review]
I am happy with these changes, just a couple of suggestions.

@HarshCasper is currently (and coincidentally) needing help on this topic, so I will let give the final approval, but LGTM

**Review by simonrw — APPROVED** [review]
This worked for @HarshCasper

File: content/en/user-guide/aws/ecs/index.md (hunk 2 above)
Comment (lukqw) [reply content/en/user-guide/aws/ecs/index.md]: Not entirely sure how I ended up with that spacing 😂

**Pull request merged**